tools/topology/sof/pipe-crossover-playback.m4

```diff
@@ -0,0 +1,81 @@
+# Crossover Volume Pipeline
+#
+#  Low Latency Playback with demux and volume.
+#
+# Pipeline Endpoints for connection are :
+#
+#	B2 (DAI buffer)
+#       Playback Crossover
+#
+# host PCM_P -- B0 --> Crossover 0 --> B1 ---> DAI0
+#                                |
+#                                pipeline n+1 --> DAI
+
+# Include topology builder
+include(`utils.m4')
+include(`buffer.m4')
+include(`pcm.m4')
+include(`crossover.m4')
+include(`bytecontrol.m4')
+
+#
+# Controls for Crossover
+#
+
+define(CROSSOVER_priv, concat(`crossover_bytes_', PIPELINE_ID))
+define(MY_CROSSOVER_CTRL, concat(`crossover_control_', PIPELINE_ID))
```

Comment:
It would be good to undefine these in the end of this macro to make sure there's every time a define for them before they are used.